windows_bat/PokemonGo-Bot-Start.bat

```diff
@@ -41,7 +41,6 @@ if '%1'=='ELEV' (del "%vbsGetPrivileges%" 1>nul 2>nul  &  shift /1)
 cd C:/Python27/PokemonGo-Bot/
 git pull
 git submodule update --init --recursive
-git submodule foreach git pull origin master

```

Comment:
Also, installing using the Master branch also causes the map to not work. The only way for the map to work properly is pull from the dev branch and not update using the line I removed.